internal/ulid/generator.go

```diff
@@ -18,12 +19,12 @@ var (
 
 // DefaultEntropy returns a reader that generates ULID entropy.
 // The default entropy function utilizes math/rand.Rand, which is not safe for concurrent use by multiple goroutines.
-// Therefore, this function employs x/exp/rand, as recommended by the authors of the library.
+// Therefore, this function employs math/rand/v2 (supersedes x/exp/rand), as recommended by the authors of the library.
 func DefaultEntropy() io.Reader {
 	entropyOnce.Do(func() {
-		seed := uint64(time.Now().UnixNano())
-		source := rand.NewSource(seed)
-		rng := rand.New(source)
+		var seed [32]byte
```

Comment:
The same here, `math/rand` is fine and then have: `rng := rand.New(rand.NewSource(time.Now().UnixNano()))`.